doc/whats_new.rst

```diff
@@ -12,6 +12,56 @@ Version 0.18
 Changelog
 ---------
 
+.. _model_selection_changes:
+
+Model Selection Enhancements and API Changes
+--------------------------------------------
+
+  - **The ``model_selection`` module**
+
+    The classes and functions from the formerly :mod:`sklearn.cross_validation`,
+    :mod:`sklearn.grid_search` and :mod:`sklearn.learning_curve` are
+    grouped together into a single module available at
+    :mod:`sklearn.model_selection`. There are some major changes to these
+    newly moved classes and functions. Refer below to know about the
+    changes.
```

Comment:
I agree with this, but I also believe that the fine tuning of a changelog is the role of the people doing the release... who/whenever that is!